packages/microcosm-dom/test/presenter.test.js

```diff
@@ -540,7 +540,7 @@ describe('::teardown', function() {
     expect(renders).toBe(1)
   })
 
-  it('changes during teardown do not cause a recalculation', function() {
+  it('changes during teardown do not cause a recalculation', async function() {
```

Comment:
total nitpick, but this might read better as:
```js
it('does not recalculate when changes during teardown occur'
```

Reply:
Great suggestion!

Reply:
Done in 0e03b9ce